packages/primitives/radio/src/Root.tsx

```diff
@@ -13,44 +13,56 @@ const styles = normalizeWebStyle({
   },
 })
 
+const invisibleStyles = normalizeWebStyle({
```

Comment:
what is the `_webOnly` on line 11?

Reply:
styles only for the web, this comes from `Stilli`

Reply:
but this file is already just for the web 🤔

Reply:
?